contracts/core/OptionsPremiumPricer.sol

```diff
@@ -18,18 +18,23 @@ contract OptionsPremiumPricer {
     IVolatilityOracle public immutable volatilityOracle;
     IPriceOracle public immutable priceOracle;
     ICompToken public immutable cToken;
+    // IKEEP3rVolatility: 0xCCdfCB72753CfD55C5afF5d98eA5f9C43be9659d
+    IBlackScholes public immutable blackScholes;
```

Comment:
Would you want to just make this a library?

Reply:
Seems cleaner since it's just an immutable variable anyway

Reply:
you're referring to IBlackScholes? this is the [IKEEP3rVolatility](0xCCdfCB72753CfD55C5afF5d98eA5f9C43be9659d) contract. Do you mean copy pasting the quoteAll method from the contract and making it an internal library instead?

Reply:
Oh I see- I didn't know we were using an external contract for the black scholes calculation. I'm all for using whatever keeper has.